<ui version="4.0" >
 <class>photoTexturingDialog</class>
 <widget class="QDialog" name="photoTexturingDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>360</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>560</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>560</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Photo texturing</string>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>0</y>
     <width>541</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QGroupBox" name="groupBox" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title" >
       <string>Cameras:</string>
      </property>
      <property name="flat" >
       <bool>false</bool>
      </property>
      <property name="checkable" >
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="layoutWidget" >
       <property name="geometry" >
        <rect>
         <x>370</x>
         <y>20</y>
         <width>161</width>
         <height>221</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Textures:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="textureListWidget" />
        </item>
        <item>
         <widget class="QPushButton" name="calculateTexturesPushButton" >
          <property name="text" >
           <string>Calculate UV Coords.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="combineTexturesPushButton" >
          <property name="text" >
           <string>Combine Textures</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>20</y>
         <width>351</width>
         <height>221</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QTableWidget" name="cameraTableWidget" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="selectionBehavior" >
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="columnCount" >
           <number>2</number>
          </property>
          <column>
           <property name="text" >
            <string>Camera</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Texture</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QPushButton" name="addCameraPushButton" >
            <property name="text" >
             <string>Add Camera</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeCameraPushButton" >
            <property name="text" >
             <string>Remove Camera</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="assignImagePushButton" >
            <property name="text" >
             <string>Assign Image</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Photo texturing Configuration File:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLineEdit" name="calibrationFileLineEdit" />
        </item>
        <item>
         <widget class="QPushButton" name="configurationLoadPushButton" >
          <property name="text" >
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="configurationSavePushButton" >
          <property name="text" >
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exportToMaxScriptPushButton" >
          <property name="text" >
           <string>export to MaxScript</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QPushButton" name="applyPushButton" >
        <property name="text" >
         <string>Apply Texture to Mesh</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelPushButton" >
        <property name="text" >
         <string>Restore Original Texture</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closePushButton" >
        <property name="text" >
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
